zerver/lib/test_helpers.py

```diff
@@ -208,28 +225,36 @@ def __init__(self, post_data, user_profile, assert_callback=None):
 
 class AuthedTestCase(TestCase):
     # Helper because self.client.patch annoying requires you to urlencode
+
     def client_patch(self, url, info={}, **kwargs):
+        # type: (str, Any, Dict[str, Any]) -> HttpResponse
         info = urllib.parse.urlencode(info)
         return self.client.patch(url, info, **kwargs)
+
     def client_put(self, url, info={}, **kwargs):
+        # type: (str, Any, Dict[str, Any]) -> HttpResponse
         info = urllib.parse.urlencode(info)
         return self.client.put(url, info, **kwargs)
+
     def client_delete(self, url, info={}, **kwargs):
+        # type: (str, Any, Dict[str, Any]) -> HttpResponse
         info = urllib.parse.urlencode(info)
         return self.client.delete(url, info, **kwargs)
 
     def login(self, email, password=None):
+        # type: (str, str) -> HttpResponse
         if password is None:
             password = initial_password(email)
         return self.client.post('/accounts/login/',
                                 {'username': email, 'password': password})
 
     def register(self, username, password, domain="zulip.com"):
+        # type: (str, str, str) -> HttpResponse
         self.client.post('/accounts/home/',
                          {'email': username + "@" + domain})
         return self.submit_reg_form_for_user(username, password, domain=domain)
 
-    def submit_reg_form_for_user(self, username, password, domain="zulip.com"):
+    def submit_reg_form_for_user(self, username, password, domain="zulip.com"):  # type: (str, str, str) -> HttpResponse
```

Comment:
Interesting, it'd be great to show the mypy crash to the mypy developers, since that definitely shouldn't happen...